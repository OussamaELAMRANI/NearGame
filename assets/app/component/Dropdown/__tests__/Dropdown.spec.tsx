import { cleanup } from '@testing-library/react'
import { create } from 'react-test-renderer'
import Dropdown from '../index'

describe('Dropdown.tsx', () => {
  afterEach(cleanup)

  it('should render correctly !', () => {
    const elem = create(<Dropdown />).toJSON()
    expect(elem).toMatchSnapshot()
  })
})
