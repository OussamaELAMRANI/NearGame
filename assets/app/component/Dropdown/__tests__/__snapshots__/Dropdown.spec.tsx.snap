// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown.tsx should render correctly ! 1`] = `
<div>
   Dropdown 
</div>
`;
